src/components/MenuButtons.js

```diff
@@ -1,44 +1,123 @@
-import Link from 'next/link';
+import Link from '@material-ui/core/Link';
+import styled from '@emotion/styled';
 import {
   Button,
   List,
   ListItem,
 } from '@material-ui/core';
+import Grid from '../utils/Grid';
+import { DesktopOnly, MobileOnly } from './MediaQuery';
 
-// later will dynamically get userId from auth
-const userId = 'qui-gonj';
+const Logo = styled.div`
+grid-column: 1/6;
+font-size:1.5rem;
+`;
 
-export default function MenuButtons({ toggleMenu }) {
+const Menu = styled.div`
+grid-column-start:6;
+
+
+.link {
+    margin: 0 1rem;
+}
+`;
+
+const Profile = styled.div`
+grid-column:11/12;
+font-size: .75rem;
+justify-self: end;
+`;
+
+const Container = styled.div`
+background: #0079D3;
+color: white;
+height: 50px;
+a {
+    color: white;
+}
+`;
+
+const MobileContainer = styled.div`
+height: 100%;
+margin: 0;
+padding-top: 2rem;
+background: #0079D3;
+color: white;
+Button {
+    color: white;
+}
+`;
+
+export default function Links({ toggleMenu }) {
+  const userId = 'qui-gonj';
   return (
-    <List>
-      <ListItem>
-        <Link href={`/${userId}/prospects`}>
-          <Button onClick={toggleMenu}>
-            Prospects
-          </Button>
-        </Link>
-      </ListItem>
-      <ListItem>
-        <Link href={`/${userId}/pledges`}>
-          <Button onClick={toggleMenu}>
-            Pledges
-          </Button>
-        </Link>
-      </ListItem>
-      <ListItem>
-        <Link href="/ministry-partners">
-          <Button onClick={toggleMenu}>
-            Partners
-          </Button>
-        </Link>
-      </ListItem>
-      <ListItem>
-        <Link href="/staff">
-          <Button onClick={toggleMenu}>
-            Staff
-          </Button>
-        </Link>
-      </ListItem>
-    </List>
+
+    <>
+      {/* Mobile Menu */}
+      <MobileOnly>
+        <MobileContainer>
+          <List>
+            <ListItem>
+              <Link href={`/${userId}/prospects`}>
+                <Button onClick={toggleMenu}>
+                  Prospects
+                </Button>
+              </Link>
+            </ListItem>
+            <ListItem>
+              <Link href={`/${userId}/pledges`}>
+                <Button onClick={toggleMenu}>
+                  Pledges
+                </Button>
+              </Link>
+            </ListItem>
+            <ListItem>
+              <Link href="/ministry-partners">
+                <Button onClick={toggleMenu}>
+                  Partners
+                </Button>
+              </Link>
+            </ListItem>
+            <ListItem>
+              <Link className="link" href="/ministers">
```

Comment:
Using `className` here doesn't appear to have any effect. Was there a reason I'm just not noticing?

Reply:
I was using it for spacing, but I've taken that out and am handling it all in the base CSS so it looks much cleaner in the markup.